wellcomecollection/wellcomecollection.org
# Articles > stories URL changes

## What does this change?
For #11071

- redirects /articles/:id to /stories/:id
- redirects /articles' to /search/stories
- redirects /stories/comic to /search/stories filtered by comic

These redirects are temporarily done in the next.config.js file. This means we can deploy the code and redirects at the same time and nothing will break. Once this is live, I'll move the redirects to the lambda function we use for all the other redirects and remove these ones.

- updates 'more stories' and 'more comics' links on the story landing page to point to the stories search equivalents
- amends linkResolver to point article and webcomic types to /stories/:id
- removes the page templates that were serving /articles and /stories/comic

Also adds hotjar script for stories pages, see: #11331

Questions:
- ~is it ok to point /stories/comic to /search/stories filtered by comic?~ -yes
- ~was anything else being served at /stories/:customType?~ - no
- ~don't like having the redirects in more than one place. Can/should we use the redirection lambda rather than next config redirects?~ - yes, but temporarily use these so nothing breaks
- ~do we need to update the literalRedirects that point to articles/:uid?~ - yes, as part of the lambda work

## How to test
- need to restart the app when checking out the branch in order for the redirects to work
- visiting localhost:3000/articles/the-power-of-art should render the page at localhost:3000/stories/the-power-of-art
- visiting /articles takes you to /search/stories
- visiting /stories/comic takes you to /search/stories filtered by comic

You will also be able to see the 301 redirects in the network tab of the developer toolbar.

- Following the 'more stories' and 'more comics' links on the stories landing page takes you directly to the above search pages
- Story cards for articles/web comics all point to /stories/:uid
- Canonical link hrefs for articles/web point to /stories/:uid

## How can we measure success?

We serve pages at /stories/:uid rather than /articles:uid

## Have we considered potential risks?

Will there be an impact on Analytics data?
